modules/core/04-channel/types/packet.go

```diff
@@ -100,6 +101,42 @@ func commitV2Packet(packet Packet) []byte {
 	return hash[:]
 }
 
+// CommitPacketV2 returns the V2 packet commitment bytes. The commitment consists of:
+// sha256_hash(timeout) + sha256_hash(destinationID) + sha256_hash(packetData) from a given packet.
```

Comment:
```suggestion
// sha256_hash(timeout) + sha256_hash(destinationID) + sha256_hash(packetData) for a given packet.
```

Reply:
sorry missed this comment, will include this in a follow up!